drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```diff
@@ -524,12 +524,28 @@ config NRF70_RSSI_STALE_TIMEOUT_MS
 	  value as the driver does not store it and requires RPU to provide the
 	  information.
 
+config NRF_WIFI_GLOBAL_HEAP
+	bool "Use K_HEAP for Wi-Fi driver"
```

Comment:
`K_HEAP` is still technical, may be use `Zephyr kernel heap`

Reply:
Will change.